client/src/ui/organisms/article-actions/language-menu/index.tsx

```diff
@@ -156,13 +158,23 @@ function LocaleRedirectSetting() {
 
   return (
     <form className="submenu-item locale-redirect-setting">
-      <Switch
-        name="locale-redirect"
-        checked={locale === preferredLocale}
-        toggle={toggle}
-      >
-        Remember language
-      </Switch>
+      <div className="group">
+        <Switch
+          name="locale-redirect"
+          checked={locale === preferredLocale}
+          toggle={toggle}
+        >
+          Remember language
+        </Switch>
+        <a
+          href="https://github.com/orgs/mdn/discussions/739"
+          rel="external noopener"
+          target="_blank"
+          title="Enable this setting to automatically switch to this language when it's available. (Click to learn more.)"
+        >
+          <Icon name="question-mark" />
+        </a>
```

Comment:
Feels like, semantically, the `<a>` should go within the `<Switch>`, did you explore doing that?

Reply:
If I put it inside the `<Switch />`, then I'd have to prevent the default behavior, because what's inside there is the label for the switch, so it toggles the value.

Reply:
So I don't think it belongs in there.